### DISCUSSION TITLE
Commands confirm

### DISCUSSION THREAD

**Discussion by deanle8** [BG-Software-LLC/SuperiorSkyblock2#1671] [Pending Feature Requests]
Add the "confirm" arg for the following commands: 

/island kick player-name confirm
/island disband confirm
/island ban player-name confirm
.....
It helps to do quick confirm via command without opening the gui.

**Comment by OmerBenGera** [comment]
You can disable these menus in the config file, if I am not wrong.

**Reply by deanle8** [reply]
yes i know, but can you do my idea, cause i need to do some things with it
